lib/adapters/REST/endpoints/concept.ts

```diff
@@ -34,6 +34,26 @@ export const createWithId: RestEndpoint<'Concept', 'createWithId'> = (
   return raw.put<ConceptProps>(http, `${basePath(params.organizationId)}/${params.conceptId}`, data)
 }
 
+export const patch: RestEndpoint<'Concept', 'patch'> = (
+  http: AxiosInstance,
+  params: UpdateConceptParams,
+  data: OpPatch[],
+  headers?: RawAxiosRequestHeaders
+) => {
+  return raw.patch<ConceptProps>(
+    http,
+    `${basePath(params.organizationId)}/${params.conceptId}`,
+    data,
+    {
+      headers: {
+        'X-Contentful-Version': params.version ?? 0,
```

Comment:
why do we want to provide a default version? i think the error will be confusing in case it doesn't match